emcc.py

```diff
@@ -52,7 +52,7 @@
 # steps.  In packaged versions (e.g. emsdk) this file does not exist (because
 # it is excluded in tools/install.py) and these steps are assumed to have been
 # run already.
-if os.path.exists(utils.path_from_root('.git')) and os.path.exists(utils.path_from_root('bootstrap.py')):
+if os.path.exists(utils.path_from_root('.git')):
```

Comment:
The comment needs to be updated too, with an explanation for why this file can always be assumed to exist.

Reply:
I just reverted this line to keep this change small.